webapp/web/Dockerfile

```
@@ -0,0 +1,19 @@
+# Production deployment spec for the webapp.
+
+FROM golang:1.22.11-bookworm as builder
+
+RUN apt-get update
+RUN apt-get install -qy --no-install-suggests git sudo
+WORKDIR /go/src/wpt.fyi
+
+# Default to the existing path when running inside the development docker container.
+ARG LOCAL_SRC=wpt.fyi
+COPY $LOCAL_SRC .
+RUN make go_build
```

```suggestion
RUN CGO_ENABLED=0  make go_build
```

This will create a statically linked binary.

---

It's weird that this causes a "file not found" error and not a runtime linker error, but hey, it works.